packages/bpk-component-breakpoint/src/BpkBreakpoint.js

```diff
@@ -29,11 +31,18 @@ const BREAKPOINTS = {
   ABOVE_TABLET: TOKENS.breakpointQueryAboveTablet,
 };
 
-const BpkBreakpoint = props => (
-  <MediaQuery query={props.query}>{props.children}</MediaQuery>
-);
+type Props = {
+  children: Node | (() => mixed),
+  query: string | $Keys<typeof BREAKPOINTS>,
```

Comment:
This should take a function like it does in `propTypes`, I'll fix it.

Reply:
Ah I see what you were doing here now, and I messed it up 😅

Reply:
So one important distinction here is that I had to change it to `string | $Values<typeof BREAKPOINTS>,` (values instead of keys).

If we allow `$Keys` that evaluates to `'MOBILE' | 'TABLET' | 'TABLET_ONLY' | ..etc`, but what users will actually pass in is the values that the keys refer to: `'32.25rem' | '71.25rem' | ..etc`. `$Values` achieves that.

Fun fact, when we first adopted Flow the `$Values` type didn't exist and stuff like this was a huge headache!